### ISSUE TITLE
autofs: kernel panic on disconnection of camcorder that was represented by /autofs/da1s1/

### ISSUE THREAD

**Issue by grahamperrin** [trueos/trueos-core#357]
## Steps

1. Lumina
2. USB connection to camcorder (Sony Handycam HDR-CX410VE)
3. QTerminal
4. `cd /autofs`
5. `ls -ahl da1s1`
6. observe completion of the directory listing (empty, that's a separate issue)
7. wait a few seconds
8. disconnect the camcorder

## Expected

- continued use of the operating system

## Actual result

- sudden restart of the computer (HP EliteBook 850 G2) 

– a kernel panic, I guess.

## When reproducing the kernel panic

After starting the OS, before connecting the camcorder, be prepared to remove the debris in `/media/` and/or the mount point debris. 

## Relevant extract from /var/log/messages

```text
Mar  6 15:33:54  kernel: ugen0.13: <Sony HDR-CX410VE> at usbus0
Mar  6 15:33:54  kernel: umass0 on uhub0
Mar  6 15:33:54  kernel: umass0: <Sony HDR-CX410VE, class 0/0, rev 2.00/6.00, addr 12> on usbus0
Mar  6 15:33:54  kernel: umass0:  SCSI over Bulk-Only; quirks = 0xc000
Mar  6 15:33:54  kernel: umass0:1:0: Attached to scbus1
Mar  6 15:33:54  kernel: da0 at umass-sim0 bus 0 scbus1 target 0 lun 0
Mar  6 15:33:54  kernel: da0: <Sony Camcorder 1.00> Removable Direct Access SCSI device
Mar  6 15:33:54  kernel: da0: Serial Number 653C40342A48
Mar  6 15:33:54  kernel: da0: 40.000MB/s transfers
Mar  6 15:33:54  kernel: da0: 60906MB (124735488 512 byte sectors)
Mar  6 15:33:54  kernel: da0: quirks=0x2<NO_6_BYTE>
Mar  6 15:33:54  kernel: da1 at umass-sim0 bus 0 scbus1 target 0 lun 1
Mar  6 15:33:54  kernel: da1: <Sony Camcorder 1.00> Removable Direct Access SCSI device
Mar  6 15:33:54  kernel: da1: Serial Number 653C40342A48
Mar  6 15:33:54  kernel: da1: 40.000MB/s transfers
Mar  6 15:33:54  kernel: da1: 25MB (51200 512 byte sectors)
Mar  6 15:33:54  kernel: da1: quirks=0x2<NO_6_BYTE>
Mar  6 15:34:07  kernel: ugen0.13: <Sony HDR-CX410VE> at usbus0 (disconnected)
Mar  6 15:34:07  kernel: umass0: at uhub0, port 2, addr 12 (disconnected)
Mar  6 15:34:07  kernel: da0 at umass-sim0 bus 0 scbus1 target 0 lun 0
Mar  6 15:34:07  kernel: da0: <Sony Camcorder 1.00> s/n 653C40342A48 detached
Mar  6 15:34:07  kernel: da1 at umass-sim0 bus 0 scbus1 target 0 lun 1
Mar  6 15:34:07  kernel: da1: <Sony Camcorder 1.00> s/n 653C40342A48 detached
Mar  6 15:34:07  kernel: (da0:umass-sim0:0:0:0): Periph destroyed
Mar  6 15:34:07  kernel: (da1:umass-sim0:0:0:1): Periph destroyed
Mar  6 15:34:07  kernel: umass0: detached
Mar  6 15:34:39  kernel: ugen0.13: <Sony HDR-CX410VE> at usbus0
Mar  6 15:34:39  kernel: umass0 on uhub0
Mar  6 15:34:39  kernel: umass0: <Sony HDR-CX410VE, class 0/0, rev 2.00/6.00, addr 13> on usbus0
Mar  6 15:34:39  kernel: umass0:  SCSI over Bulk-Only; quirks = 0xc000
Mar  6 15:34:39  kernel: umass0:1:0: Attached to scbus1
Mar  6 15:34:39  kernel: da0 at umass-sim0 bus 0 scbus1 target 0 lun 0
Mar  6 15:34:39  kernel: da0: <Sony Camcorder 1.00> Removable Direct Access SCSI device
Mar  6 15:34:39  kernel: da0: Serial Number 653C40342A48
Mar  6 15:34:39  kernel: da0: 40.000MB/s transfers
Mar  6 15:34:39  kernel: da0: 60906MB (124735488 512 byte sectors)
Mar  6 15:34:39  kernel: da0: quirks=0x2<NO_6_BYTE>
Mar  6 15:34:39  kernel: da1 at umass-sim0 bus 0 scbus1 target 0 lun 1
Mar  6 15:34:39  kernel: da1: <Sony Camcorder 1.00> Removable Direct Access SCSI device
Mar  6 15:34:39  kernel: da1: Serial Number 653C40342A48
Mar  6 15:34:39  kernel: da1: 40.000MB/s transfers
Mar  6 15:34:39  kernel: da1: 25MB (51200 512 byte sectors)
Mar  6 15:34:39  kernel: da1: quirks=0x2<NO_6_BYTE>
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:58  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:58  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:58  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:58  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:58  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:34:58  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:34:58  kernel: flags (VI_ACTIVE)
Mar  6 15:34:58  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:34:58  kernel: lock type devfs: UNLOCKED
Mar  6 15:34:58  kernel: dev da1s1
Mar  6 15:34:58  automountd[5012]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5021, terminated with exit status 1
Mar  6 15:34:58  automountd[5012]: mount failed
Mar  6 15:34:58  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:34:59  automountd[5025]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5036, terminated with exit status 1
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:59  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:59  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:34:59  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:34:59  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:34:59  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:34:59  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:34:59  kernel: flags (VI_ACTIVE)
Mar  6 15:34:59  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:34:59  kernel: lock type devfs: UNLOCKED
Mar  6 15:34:59  kernel: dev da1s1
Mar  6 15:34:59  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:34:59  automountd[5025]: mount failed
Mar  6 15:35:01  automountd[5043]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5052, terminated with exit status 1
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:35:01  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:35:01  kernel: flags (VI_ACTIVE)
Mar  6 15:35:01  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:35:01  kernel: lock type devfs: UNLOCKED
Mar  6 15:35:01  kernel: dev da1s1
Mar  6 15:35:01  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:35:01  automountd[5043]: mount failed
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:01  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:01  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:01  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:35:01  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:35:01  kernel: flags (VI_ACTIVE)
Mar  6 15:35:01  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:35:01  kernel: lock type devfs: UNLOCKED
Mar  6 15:35:01  kernel: dev da1s1
Mar  6 15:35:01  automountd[5054]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5063, terminated with exit status 1
Mar  6 15:35:01  automountd[5054]: mount failed
Mar  6 15:35:01  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:35:02  automountd[5067]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5076, terminated with exit status 1
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:02  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:02  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:02  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:02  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:02  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:35:02  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:35:02  kernel: flags (VI_ACTIVE)
Mar  6 15:35:02  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:35:02  kernel: lock type devfs: UNLOCKED
Mar  6 15:35:02  kernel: dev da1s1
Mar  6 15:35:02  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:35:02  automountd[5067]: mount failed
Mar  6 15:35:03  automountd[5080]: "mount -t msdosfs -o nosuid,async,-M=777,-o=large,nosuid,noatime,automounted /dev/da1s1 /autofs/da1s1/", pid 5090, terminated with exit status 1
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:03  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:03  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): WRITE(10). CDB: 2a 20 00 00 00 04 00 00 08 00 
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): CAM status: SCSI Status Error
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI status: Check Condition
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): SCSI sense: DATA PROTECT asc:27,0 (Write protected)
Mar  6 15:35:03  kernel: (da1:umass-sim0:0:0:1): Error 13, Unretryable error
Mar  6 15:35:03  kernel: g_vfs_done():da1s1[WRITE(offset=512, length=4096)]error = 13
Mar  6 15:35:03  kernel: fsync: giving up on dirty 0xfffff8032c9b7b10: tag devfs, type VCHR
Mar  6 15:35:03  kernel: usecount 1, writecount 0, refcount 10 mountedhere 0xfffff80021cc1e00
Mar  6 15:35:03  kernel: flags (VI_ACTIVE)
Mar  6 15:35:03  kernel: v_object 0xfffff8032c5cfd20 ref 0 pages 7 cleanbuf 7 dirtybuf 1
Mar  6 15:35:03  kernel: lock type devfs: UNLOCKED
Mar  6 15:35:03  kernel: dev da1s1
Mar  6 15:35:03  kernel: WARNING: autofs_trigger_one: request for /autofs/da1s1/ completed with error 5
Mar  6 15:35:03  automountd[5080]: mount failed
Mar  6 15:35:09  kernel: ugen0.13: <Sony HDR-CX410VE> at usbus0 (disconnected)
Mar  6 15:35:09  kernel: umass0: at uhub0, port 2, addr 13 (disconnected)
Mar  6 15:35:09  kernel: da0 at umass-sim0 bus 0 scbus1 target 0 lun 0
Mar  6 15:35:09  kernel: da0: <Sony Camcorder 1.00> s/n 653C40342A48 detached
Mar  6 15:35:09  kernel: da1 at umass-sim0 bus 0 scbus1 target 0 lun 1
Mar  6 15:35:09  kernel: da1: <Sony Camcorder 1.00> s/n 653C40342A48 detached
Mar  6 15:35:09  kernel: (da1:umass-sim0:0:0:1): Periph destroyed
Mar  6 15:35:09  kernel: (da0:umass-sim0:0:0:0): Periph destroyed
Mar  6 15:35:09  kernel: umass0: detached
Mar  6 15:36:29  syslogd: kernel boot file is /boot/kernel/kernel
```

**Comment by grahamperrin** [comment]
https://discourse.trueos.org/t/new-autofs-mounting/878/43?u=grahamperrin: 

> … You've read how the new automount system works, correct?

Yes. As quoted by me in the topic. 

> … mounts when you cd into the directory. 

Not when the directory is as shown at step 4.

**Comment by beanpole135** [comment]
Please keep responses to question within the same communication channel. This comment has no relevance to the ticket itself.

**Comment by beanpole135** [comment]
Closed as a duplicate of #359 
(since 359 seems to have slightly more information even though it came later)
